icechunk/src/session.rs

```diff
@@ -108,6 +109,8 @@ pub enum SessionErrorKind {
     InvalidIndex { coords: ChunkIndices, path: Path },
     #[error("invalid chunk index for splitting manifests: {coords:?}")]
     InvalidIndexForSplitManifests { coords: ChunkIndices },
+    #[error("incompatible manifest splitting config when merging sessions")]
+    IncompatibleSplits,
```

Comment:
Can we provide more information here? Maybe the first two non compatible splits we detected (or all of them)

Reply:
I am now reporting the splitting configs for the two sessions. It seems hard to actual report a Path for the node whose splits are incompatible.